com.unity.visualeffectgraph/Shaders/Sort.compute

```diff
@@ -290,15 +310,14 @@ void MERGE_PASS(uint3 groupId           : SV_GroupID,
     } while (!done);
 
     // Select left or right: Must handle special case for equality
-    bool select0 = (key0 == key1 && (id & 1)) || (CompareKeys(key0,key1) != reverse);
-    uint value = inputSequence[(select0 ? index0 : index1)].value;
+    bool select0 = (kvp0.key == kvp1.key && kvp0.value == kvp1.value && (id & 1)) || (CompareKVP(kvp0, kvp1) != reverse);
```

Comment:
The first condition does not seem relevant anymore. CompareKVP test seems enough